python/tskit/tables.py

```diff
@@ -1635,7 +1669,9 @@ def append_columns(self, metadata=None, metadata_offset=None):
         :type metadata_offset: numpy.ndarray, dtype=np.uint32.
         """
         self.ll_table.append_columns(
-            dict(metadata=metadata, metadata_offset=metadata_offset)
+            dict(
```

Comment:
we should be able to just leave ``metadata_schema`` out of this, right?

Reply:
We can't as this is calling the low-level method.

Reply:
Should be OK now, right?

Reply:
Yep gone in https://github.com/tskit-dev/tskit/pull/687/commits/682b97da754cb07b00969d8d75c6b06d0314976f